src/main/java/com/insightfullogic/honest_profiler/ports/console/ConsoleLogDumpApplication.java

```diff
@@ -0,0 +1,179 @@
+package com.insightfullogic.honest_profiler.ports.console;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.kohsuke.args4j.CmdLineException;
+import org.kohsuke.args4j.CmdLineParser;
+import org.kohsuke.args4j.Option;
+
+import com.insightfullogic.honest_profiler.core.Conductor;
+import com.insightfullogic.honest_profiler.core.Monitor;
+import com.insightfullogic.honest_profiler.core.parser.LogEventListener;
+import com.insightfullogic.honest_profiler.core.parser.Method;
+import com.insightfullogic.honest_profiler.core.parser.StackFrame;
+import com.insightfullogic.honest_profiler.core.parser.TraceStart;
+import com.insightfullogic.honest_profiler.ports.sources.FileLogSource;
+
+public class ConsoleLogDumpApplication {
+
+    private final Console output;
+    private final Console error;
+
+    private File logLocation;
+    
+
+    private boolean sfQuiet=false;
+    private boolean mQuiet=true;
+
+    public static void main(String[] args) {
+        ConsoleLogDumpApplication entry = new ConsoleLogDumpApplication(() -> System.err, () -> System.out);
+        CmdLineParser parser = new CmdLineParser(entry);
+
+        try {
+            parser.parseArgument(args);
+            entry.run();
+        } catch (CmdLineException e) {
+            System.err.println(e.getMessage());
+            parser.printUsage(System.err);
+        }
+    }
+
+    public ConsoleLogDumpApplication(final Console error, final Console output) {
+        this.output = output;
+        this.error = error;
+    }
+
+    @Option(name = "-log", usage = "set the log that you want to parser or use", required = true)
+    public void setLogLocation(String logLocation) {
+        setLogLocation(new File(logLocation));
+    }
+    
+    @Option(name = "-qsf", usage = "set stack frame logging to quiet", required = false)
```

Comment:
I would probably prefer more human readable option names, rather than short ones. Typing isn't the bottleneck.